tracks/r/exercises/phone-number/mentoring.md

```diff
@@ -0,0 +1,63 @@
+### Reasonable solution
+
+This is a straight-forward, logical solution:
+
+```r
+parse_phone_number <- function(number_string) {
+  number_string <- gsub("[^[:digit:]]", "", number_string)
+
+  digits <- nchar(number_string)
+  if (digits < 10 || digits > 11)
+    return(NULL)
+  if (digits == 11) {
+    if (!grepl("^1", number_string)) {
+      return(NULL)
+    } else {
+      number_string <- substr(number_string, 2, 11)
+    }
+  }
+
+  if (!grepl("^[2-9].{2}[2-9]", number_string)) {
+    return(NULL)
+  }
+
+  return(number_string)
+}
+```
+
+
+### Common suggestions
+
+- Just like with Word Count, `[regex](https://regexr.com/)` is a great resource for visualizing regular expression pattern matching.
```

Comment:
Is Word Count the exercise, or a website, or a tool in r? (Not clear from this description)